### DISCUSSION TITLE
Error when using PonyXL Models

### DISCUSSION THREAD

**Discussion by aeternitas97** [vladmandic/sdnext#3291] [Q&A]
Hi all,

Been using this successfully with JuggernautX for a while. Tried to use PonyXL and a variant (EverclearPNY) today, but can't get them to work. 

The image generates fine up until the end. I can see an image generating in the preview window, but it changes to a totally black image on the last generation step. This followed by the following console message:

"Failed to validate samples: sample=(768, 768, 3) min=0.00 max=0.00 mean=0.00 invalid=1769472"

I am using zLuda, if that affects anything.

**Comment by vladmandic** [comment]
which probably means that particular model has non-fixed vae built-in (non-fixed vae that requires fp32 since it overflows in fp16) and you need to enable vae upcast in settings.
silly to see models using non-fixed vae nowadays.

**Comment by brknsoul** [comment]
You could alternatively grab this 16bit VAE https://civitai.com/models/140686?modelVersionId=155933
place it in `models\vae` and enable it in settings.

**Comment by HamOntG** [comment]
> Hi all,
> 
> Been using this successfully with JuggernautX for a while. Tried to use PonyXL and a variant (EverclearPNY) today, but can't get them to work.
> 
> The image generates fine up until the end. I can see an image generating in the preview window, but it changes to a totally black image on the last generation step. This followed by the following console message:
> 
> "Failed to validate samples: sample=(768, 768, 3) min=0.00 max=0.00 mean=0.00 invalid=1769472"
> 
> I am using zLuda, if that affects anything.

For what it's worth, I got this exact error when I was using an SDXL VAE for an SD 1.5 model.
